packages/mask/src/components/DataSource/useNextID.ts

```diff
@@ -66,8 +71,15 @@ export function useNextIDConnectStatus() {
             persona: currentPersona?.identifier.toText(),
         })
         isOpenedVerifyDialog = true
+        isOpenedFromButton = false
         return false
-    }, [username, enableNextID, lastStateRef.value])
+    }, [username, enableNextID, lastStateRef.value, isOpenedVerifyDialog, isOpenedFromButton])
```

Comment:
Is it necessary to depend `isOpenedVerifyDialog` and  `isOpenedFromButton`?

Reply:
it ls necessary to depend `isOpenedVerifyDialog`